WebView/app/src/main/assets/index.html

```diff
@@ -16,18 +16,27 @@
 
 <!DOCTYPE html>
 <html>
-    <head>
-        <link href="app.css" rel="stylesheet">
-        <title></title>
-    </head>
-    <body onload="getData()">
-        <h1 id="location">Today's Weather</h1>
-        <h2 class="day">Monday</h2><img id= "icon" alt="weather" class="center" src="https://gcoleman799.github.io/res/drawable/sunny.png">
-        <h2 id="description">Sunny and 75</h2>
-        <form>
-            <input class="button" onclick="sendAndroidMessage()" type="button" value="Send Message">
-        </form>
-        <script src="main.js">
-        </script>
-    </body>
+<head>
+    <link href="app.css" rel="stylesheet">
+    <title></title>
```

Comment:
Should there be a title?

Reply:
No, that is an artifact of an older version. I have removed the empty tag.